core/bootstrap/src/main/java/org/wildfly/swarm/bootstrap/modules/ContainerModuleFinder.java

```diff
@@ -89,6 +89,40 @@ public void buildModule(ModuleSpec.Builder builder, ModuleLoader delegateLoader)
                         null,
                         ModuleIdentifier.create("javax.enterprise.api"), false));
 
+        builder.addDependency(
```

Comment:
If we want overrides, we may need to provide hooks similar to this one: https://github.com/jboss-modules/jboss-modules/blob/1.x/src/main/java/org/jboss/modules/Main.java#L391